source/ScreensaverHero.hx

```diff
@@ -0,0 +1,36 @@
+package;
+
+import flixel.FlxG;
+import flixel.FlxSprite;
+
+using flixel.util.FlxSpriteUtil;
+
+// Part 2: Added a Screensaver Hero Class
+class ScreensaverHero extends FlxSprite
+{
+	// Implement Constructor
+	public function new()
+	{
+		// Invoke parent class constructor
+		super(x, y);
+		this.x = x;
+		this.y = y;
+		// Make a simple rectangle sprite
+		var myRectangle = new FlxSprite(x, y);
+		makeGraphic(30, 10, true);
```

Comment:
Why did you choose to include `true` as an argument to this method?

Reply:
When I was reading about makeGraphic() the unique parameter allows the graphic to be a unique instance in the graphics cache, which allows you to modify the pixels of a sprite without modifying other sprites with the same "bitmap data". I am not sure I used that correctly or for this assignment it is unnecessary?

Reply:
Gotcha, that makes sense.

You're correct that making the graphic unique protects the instances from accidentally modifying the underlying bitmap data, but setting `color` isn't actually changing that so unique ends up being unnecessary.

The magic of graphics pipelines is mostly outside of the scope of our class, but you can think of the `colorTransform()` method as a sort of bonus filter that Haxeflixel applies over that specific `ScreensaverHero` when it's time to draw it to the screen.

It's possible to do other things that actually modify the underlying graphic (in which case keeping them unique is probably desirable) but the color tinting isn't one of them.